Umbraco-Cloud/Getting-Started/The-Umbraco-Cloud-Portal/Organizations/index.md

```diff
@@ -0,0 +1,15 @@
+# Organizations 
+On Umbraco Cloud it is possible to be part of an organization. And organization is a way for you to associate users to one organization.
+The benefits are currently limited, but we are working on improving the features available for organizations.
+Currently organizations are created by Umbraco HQ - so **reach out to support** to hear more about the feature, and weather they apply for your scenario.
+
+## Invite organization to collaborate
+Once you have an organization you will be able to invite other organizations to collaborate for you.
+By inviting an organization the following is will happen
```

Comment:
Type here:
By inviting an organization the following **is** will happen
Should be:
By inviting an organization the following will happen